lib/zendesk_api/middleware/response/logger.rb

```diff
@@ -18,7 +18,10 @@ def call(env)
           @logger.debug dump_debug(env, :request_headers)
 
           @app.call(env).on_complete do |env|
-            @logger.info("Status #{env[:status].to_s}")
+            info = "Status #{env[:status].to_s}"
+            info.concat(" #{env[:body].inspect}") if (400..499).cover?(env[:status].to_i)
```

Comment:
should be careful here with logging too big things (crashes logging system / is mostly useless anyway since it's html) ... maybe just do to_s and then the first 1000 characters ?

Reply:
I did not care that so I think so too.

Reply:
fixed in e434367